bitnami/thanos/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 2.5.3
+version: 2.5.4
```

Comment:
Could you please bump the minor version as this is introducing new input parameters?
```suggestion
version: 2.6.0
```

Reply:
Done